include/igl/cr_vector_curvature_correction.cpp

```diff
@@ -0,0 +1,185 @@
+// This file is part of libigl, a simple c++ geometry processing library.
+//
+// Copyright (C) 2020 Oded Stein <oded.stein@columbia.edu>
+//
+// This Source Code Form is subject to the terms of the Mozilla Public License
+// v. 2.0. If a copy of the MPL was not distributed with this file, You can
+// obtain one at http://mozilla.org/MPL/2.0/.
+
+#include "cr_vector_curvature_correction.h"
+
+#include "orient_halfedges.h"
+#include "gaussian_curvature.h"
+
+#include "squared_edge_lengths.h"
+#include "doublearea.h"
+#include "boundary_loop.h"
+#include "internal_angles.h"
+
+#include "PI.h"
+
+
+template <typename DerivedV, typename DerivedF, typename DerivedE,
+typename DerivedOE, typename ScalarK>
+IGL_INLINE void
+igl::cr_vector_curvature_correction(
```

Comment:
So we already have `crouzeix_raviart_massmatrix` and `crouzeix_raviart_cotmatrix`. I guess you want to abbreviate that into `cr_` to keep name shorter, but I just want to make sure we're all ok with introducing an inconsistency here?